prime-angular-frontend/src/app/modules/site-registration/pages/organization-name-page/organization-name-page.component.ts

```diff
@@ -138,6 +139,22 @@ export class OrganizationNamePageComponent extends AbstractEnrolmentPage impleme
         this.orgBookOrganizations = response.results.map(o => o.names[0]?.text).filter(o => o);
         this.orgBookTotalResults = response.total;
       });
+
+    if (this.organizationId === 0) {
+      this.formState.name.addAsyncValidators(asyncValidator(this.checkOrganizationName$(), 'duplicate'));
+    }
```

Comment:
Because I can update an Org name to a duplicate, I get this situation:
![image](https://github.com/bcgov/moh-prime/assets/7586665/5d2c3c1d-9a92-41a4-bc87-17cede0a1b51)

Reply:
This should be fixed.